flow/connectors/clickhouse/clickhouse.go

```diff
@@ -104,15 +109,12 @@ func NewClickHouseConnector(
 			Provider:   credentialsProvider,
 			BucketPath: awsBucketPath,
 		},
+		chVersion: clickHouseVersion.Version.String(),
```

Comment:
Can't store 
```suggestion
		chVersion: clickHouseVersion.Version,
```
why does this need to be stored as string?

Reply:
it doesn't, we can store it as Version :)